portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -149,17 +160,26 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     commentEntity.setProperty("text", request.getParameter("comment"));
     commentEntity.setProperty("date", DateFor.format(new Date()));
     commentEntity.setProperty("timestamp", System.currentTimeMillis());
-    // Get the URL of the image that the user uploaded to Blobstore.
-    String imageUrl = getUploadedFileUrl(request, response, "image");
-    if (imageUrl != null) {
-      commentEntity.setProperty("imageURL", imageUrl);
+    // Get the URL of the image that the user uploaded to Blobstore, if it exists.
+    BlobKey blobKey = getBlobKey(request, response, "image");
+    if (blobKey != null) {
+      //String imageUrl = getUploadedFileUrl(blobKey);
```

Comment:
Avoid submitting commented code like the plague.

git has full version control so you can look up previous implementation if you're so inclined.

Reply:
Done